exporttestconfiguration.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can we move these json files to ./test/configuration?

Reply:
Since moving the testauthenvironment.json might not be straightforward (might have to update corresponding PS scripts that use the file), it think it would be better to do it in a separate follow-up PR. Thoughts?

Reply:
Is it possible at least to move the cors and export configuration.json file into ./test/configuration as suggested? That way, we can also update the build script to copy the folder as opposed to individual files. We can then migrate the testauthenvironment.json later?

Reply:
Moved it to test/configuration and updated the build pipeline